### PR TITLE
Rename price -> spot_price, fix rubocop

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

**Comment by mwrock** [comment]
:+1:

File: lib/kitchen/driver/ec2.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ class Ec2 < Kitchen::Driver::Base # rubocop:disable Metrics/ClassLength
       end
       default_config :private_ip_address, nil
       default_config :iam_profile_name,   nil
-      default_config :price,              nil
+      default_config :spot_price,              nil
```
Comment (tas50) [inline lib/kitchen/driver/ec2.rb]: nitpick here, but it's not aligned anymore

**Comment by tas50** [comment]
spotprice is the right thing to do, but we need to bump this a major for this one since it's going to break anyone using price.

**Comment by jkeiser** [comment]
@tas50 `price` is an entirely unreleased change :) But we're bumping major anyway.

**Comment by proffalken** [comment]
+1 here (for the change and the version bump...)

**Pull request merged**